CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Unreleased
 
 - Support disk usage reporting (using `df`) on Alpine Linux.
+- Don't report inodes for mountpoints that report no usage percentage.
```

Comment:
I'm suggesting a wording but I'm not sure if it's any clearer 🤔 What I want is to point out what is it we've fixed, if that makes sense -- not _how_ it's fixed.

```suggestion
- When an inode has no usage percentage, report inode information successfully for the inodes that do have an usage percentage.
```

Reply:
```suggestion
- When a disk mountpoint has no inodes usage percentage, skip the mountpoint, and report the inodes information successfully for the inodes that do have an inodes usage percentage.
```